utils_test/fetch_price_data_test.go

```diff
@@ -0,0 +1,18 @@
+package utils_test
+
+import (
+	"testing"
+	"uniswap-v3-pool-watcher-bot/utils"
+)
+
+func TestFetchTokenPrice(t *testing.T) {
+	slug := "ethereum"
+	expected := 2.887
```

Comment:
Unfortunately ethereum price keeps changing so this test will probably fail. There's no need to check expected price for this test I think.